code/modules/mob/living/carbon/human/death.dm

```diff
@@ -28,7 +28,8 @@
 
 	dizziness = 0
 	jitteriness = 0
-
+	if(client && !suiciding)

```

Comment:
time to farm this as Sec by repeatedly defibbing a volunteer

Reply:
you might just want to count the number of ghosts who are hanging around at the point when the space cops show up and make the number be based off of that

Reply:
This still needs to be resolved somehow

Reply:
i'll add a check if they've died once before then